stdnum/za/__init__.py

```diff
@@ -19,3 +19,6 @@
 # 02110-1301 USA
 
 """Collection of South Africa numbers."""
+
+# provide aliases
+from stdnum.za import tin as vat  # noqa: F401
```

Comment:
According to https://support.upwork.com/hc/en-us/articles/15157294433683-South-Africa-VAT-for-Clients the VAT number is a 10 digit number starting with a 4. The existing TIN is also 10 digits but can't start with a 4.